### PR TITLE
Misc. documentation fixes

### PR DESCRIPTION
**Pull request by beeankha**
### Checklist - did you ...

- [ ] ~Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?~
     
- [ ] ~Add / update necessary tests?~
- [x] Add / update outdated documentation?

### CODE REVIEW

**Comment by jezdez** [comment]
@beeankha Was this ready for review?

**Comment by beeankha** [comment]
@jezdez I plan to add more to this PR, but if you think I should do that in a separate one, I can change this out of draft mode!

File: docs/source/concepts/recipe.rst [hunk 1]
```diff
@@ -182,35 +190,25 @@ Output metadata
...
-  * Package name
-  * Package version
-  * Descriptive metadata
-  * Where to obtain source code
-  * How to test the package
+#. Reach out to the repodata containing the dependencies for the package(s) you are installing.
+#. Determine the correct dependencies.
+#. Install a list of additional packages determined by those dependencies.
+#. For each dependency package being installed:
+   #. Unpack the tarball to look at the information contained within.
```
Comment (kathatherine) [inline docs/source/concepts/recipe.rst:204]: Wasn't sure about this, but was assuming that the last three things are done for each package, yes?

File: docs/source/user-guide/getting-started.rst [hunk 2]
```diff
@@ -14,12 +14,18 @@ collection helpful.
...
+- `Miniconda or Anaconda <https://docs.anaconda.com/anaconda/install/>`_
+- conda-build
+- Git
+
+The most straightforward way to do this is to install Miniconda or 
+Anaconda, which contain conda, and then using conda to install conda-build 
+and Git. Make sure you install these packages into a new environment
+and not your base environment.::
+
+    conda create -n my-conda-build-environment conda-build git
```
Comment (kathatherine) [inline docs/source/user-guide/getting-started.rst:28]: We could just do a `conda install conda-build git` as the suggested command, but wanted to be clear that they should make a new environment.

**Comment by beeankha** [comment]
pre-commit.ci autofix

**Comment by beeankha** [comment]
pre-commit.ci autofix

File: docs/source/concepts/package-naming-conv.rst [hunk 3]
```diff
@@ -5,67 +5,33 @@ Package naming conventions
...
+    hyphens, or dots. In usage documentation, these are referred to
+    by ``package_name``.
+
+**Package version**
+    A version number or string, often similar to ``X.Y`` or
+    ``X.Y.Z``, but it may take other forms as well.
+
+**Build string**
+    An arbitrary string that identifies a particular build of a
+    package for conda.  It may contain suggestive mnemonics, but
```
Comment (kathatherine) [inline docs/source/concepts/package-naming-conv.rst:21]:
```suggestion
    package for conda. It may contain suggestive mnemonics, but
```

Comment (kathatherine) [reply docs/source/concepts/package-naming-conv.rst:21]: Those double-spaces are hard to spot! No idea if it affects output, though.

File: docs/source/user-guide/getting-started.rst [hunk 4]
```diff
@@ -14,12 +14,18 @@ collection helpful.
...
+Before starting the tutorials, you need to install:
 
-After you've installed Miniconda or Anaconda, you can use conda
-to install conda-build and Git.
+- `Miniconda or Anaconda <https://docs.anaconda.com/anaconda/install/>`_
+- conda-build
+- Git
+
+The most straightforward way to do this is to install Miniconda or
+Anaconda, which contain conda, and then using conda to install conda-build
```
Comment (kathatherine) [inline docs/source/user-guide/getting-started.rst:24]:
```suggestion
Anaconda, which contain conda, and then use conda to install conda-build
```

**Review by kathatherine — DISMISSED** [review]
Two more tiny tiny things and then I think we're good.

**Review by kathatherine — APPROVED** [review]
LGTM!

**Pull request merged**